### ISSUE TITLE
History unavailable when creating new notes from different devices 

### ISSUE THREAD

**Issue by AtrumGeost** [Automattic/simplenote-electron#2961]
When creating a new note using the Simplenote Electron 2.15.0-beta1 the history is not available on all devices.

### Expected
For the history option to be available everywhere.

### Observed
The history option is not consistently visible on all devices:

**Example of note created in macOS Electron App:**

_Electron macOS_
<img width="644" alt="Screenshot on 2021-07-13 at 14-33-52" src="https://user-images.githubusercontent.com/3696121/125507159-319cae78-2938-4226-90b4-d1e51e04913f.png">
_Simplenote Android_
![screenshot_20210713-143334](https://user-images.githubusercontent.com/3696121/125507172-56e32bca-6252-443b-883b-e7fc396b80cd.png)

**Example of note created with Windows Electron app:**
_Windows_
![2021-07-13 13 41 58](https://user-images.githubusercontent.com/3696121/125507498-ed5287c7-7abc-4d69-aedf-8bb84b06be50.jpg)
_Staging web app_
![Screen Shot on 2021-07-13 at 13:41:48](https://user-images.githubusercontent.com/3696121/125507635-8053efbe-2a68-4d41-a48c-52274d69816a.png)

### Reproduced

1. Create a new note using the Electron app 
2. Add some content
3. Check the history option
4. Check the history option on a different device

### Where did you see the bug

- System Make:
- System Model:
- OS: macOS Big Sur 11.4 (20F71)  | Windows 10 20H2 | Android is on 2.18 
- Browser (if applicable): Google Chrome Version 91.0.4472.114 (Official Build) (x86_64)
- Simplenote app version:  Simplenote Electron 2.15.0-beta1

**Comment by tiagomar** [comment]
This issue also happens when collaborating with other users.

1. User A creates and edits a note
2. User A adds User B as a collaborator
    - Expected: User A AND User B have access to note's history
    - Observed: Only User B has access to note's history 

OS: macOS BigSur 11.5.2
Browsers: Chrome 93.0.4577.63 | Firefox 91.0.2 | Safari 14.1.2
Simplenote web

**Comment by codebykat** [comment]
Huh, that's an interesting one. I'm not sure if I expect the history to be shared as well as the note. If changes are made going forward, is the history shared and accessible to both parties?

**Comment by tiagomar** [comment]
Yeah, `User A` will have full history and `User B` will have all the changes from the collaboration forward, including changes from `User A`.

**Comment by BrajBliss** [comment]
+1 on the Web version. Android history works just fine.
